src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -2057,6 +2059,8 @@ def update_sync_policy_group(self, group_id: str, status: str, bucket_name: str
                                          http_status_code=500, component='rgw')
         except SubprocessError as error:
             raise DashboardException(error, http_status_code=500, component='rgw')
+        if not bucket_name:
```

Comment:
let's apply the above suggestion everywhere and introduce a param to decide whether to update_period or not and default it to False.